types/tx_msg.go

```diff
@@ -32,9 +32,12 @@ type Msg interface {
 
 // Transactions objects must fulfill the Tx
 type Tx interface {
-
-	// Gets the Msg.
+	// Gets the all the transaction's messages.
 	GetMsgs() []Msg
+
+	// ValidateBasic does a simple and lightweight validation check that doesn't
+	// require access to any other information.
+	ValidateBasic() Error
```

Comment:
I know that it's been quite conventional so far to call this `ValidateBasic`. Though I've never seen any other function being called `ValidateAdvanced` :)

So why not naming this `Validate`?

Reply:
Lol...touché. Perhaps for another PR?

Reply:
Yeah definitely not a blocking issue :+1: